docs/whats-new/release-notes/etendo-classic/translation-bundles/essentials-extensions-es/release-notes.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Release Notes
+tags:
+    - Release Notes
+    - Essentials
+    - Translations
+    - Bundle
+    - Versions
+---
+
+:octicons-package-16: Javapackage: `com.etendoerp.essentials.extensions.es_es`
+
+:material-store: Etendo Marketplace:  [Essentials Extensions Bundle ES](https://marketplace.etendo.cloud/#/product-details?module=0CE701616F5A446C83FA26633F14AE00){target="_blank"}
+
+:octicons-project-symlink-16: Original Bundle Release Notes: [Essentials Extensions Bundle](/whats-new/release-notes/etendo-classic/bundles/essentials-extensions/release-notes/)
+
+## Overview
+
+| Version | Publication Date | Compatibility With Essentials Extensions | Status | GitHub |
+| ---     |       ---        |                  ---                    | :----: | :----: |
+| [1.1.0](/whats-new/release-notes/etendo-classic/translation-bundles/essentials-extensions-es/release-notes/#110) | 16/01/2024 | [1.0.0, latest] | CS | :white_check_mark: |
```

Comment:
here you can directly define it as #110 without link

Reply:
@isaiasb-etendo Done